README.md

```diff
@@ -0,0 +1,35 @@
+# System Containers
+
+As part of our effort to reduce the number of packages that are shipped with
+the Atomic Host image, we faced the problem of how to containerize services
+that are needed to be run before a container runtime, like the upstream docker
+daemon, is running. The result: *system containers*: a way to run containers
+in production using read only images.
+
+A system container is a container that is executed out of an systemd unit file
+early in boot, using runc. The specified IMAGE must be a system image
+already fetched. If it is not already present, atomic will attempt to fetch it
+assuming it is an oci image. Installing a system container consists of
+checking it the image by default under /var/lib/containers/atomic/ and
```

Comment:
I am not crazy about the wording of this, so I fixed it up  a little, I just wanted to get something in to describe it.  We will need some licensing stuff and some description of how to build the images.